<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeLayout</class>
 <widget class="QWidget" name="MakeLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KScope - Make</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Root Directory:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Command:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="KUrlRequester" name="m_pRootURL"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="KHistoryComboBox" name="m_pCommandHistory"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget2">
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="KTextBrowser" name="m_pOutputBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Errors a&amp;nd Warnings</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QueryView" name="m_pErrorView" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>520</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pMakeButton">
       <property name="text">
        <string>&amp;Make</string>
       </property>
       <property name="shortcut">
        <string>Alt+M</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pStopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Stop</string>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCloseButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QueryView</class>
   <extends>QWidget</extends>
   <header>../../queryview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_pMakeButton</tabstop>
  <tabstop>m_pStopButton</tabstop>
  <tabstop>m_pCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>MakeLayout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pMakeButton</sender>
   <signal>clicked()</signal>
   <receiver>MakeLayout</receiver>
   <slot>slotMake()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pStopButton</sender>
   <signal>clicked()</signal>
   <receiver>MakeLayout</receiver>
   <slot>slotStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
